doc/classes/ProjectSettings.xml

```diff
@@ -228,6 +228,10 @@
 		<member name="application/config/use_custom_user_dir" type="bool" setter="" getter="" default="false">
 			If [code]true[/code], the project will save user data to its own user directory (see [member application/config/custom_user_dir_name]). This setting is only effective on desktop platforms. A name must be set in the [member application/config/custom_user_dir_name] setting for this to take effect. If [code]false[/code], the project will save user data to [code](OS user data directory)/Godot/app_userdata/(project name)[/code].
 		</member>
+		<member name="application/config/version" type="String" setter="" getter="" default="&quot;1.0.0&quot;">
+			The project's human-readable version identifier. This should always be set to a non-empty string, as some exporters rely on this value being defined.
+			[b]Note:[/b] When exporting for iOS, this version number must be incremented every time a new version is submitted to the App Store, even if the previous version was rejected by Apple. For instance, version [code]1.2.3[/code] can be incremented to [code]1.2.4[/code], [code]1.3.0[/code] or [code]2.0.0[/code].
```

Comment:
That's not entirely correct. 
If submitted application gets rejected it's enough to change/increase `CFBundleVersion`. This value doesn't get displayed to user on AppStore, `CFBundleShortVersionString` does. 
In most cases `CFBundleVersion` is used as build number, so displaying application version like `CFBundleShortVersionString (CFBundleVersion)` results in something like `1.0.0 (2520)`. And on TestFlight builds are grouped by `CFBundleShortVersionString` value.